layouts/shortcodes/3rdparty-content.html

```diff
@@ -0,0 +1,3 @@
+<blockquote class="note">
+  <div><strong>{{ T "note" }}</strong> {{ T "3rdparty_message" | safeHTML }}</div>
+</blockquote>
```

Comment:
How about:
```suggestion
<blockquote class="note 3rdpartylinks">
```

_or_, if this is always exactly a note shortcode, can we use shortcode nesting?

I'd make the shortcode name declare _what_ it is, eg: `3rdparty-content-notice`.

Reply:
Shortcode nesting it is :)